molgenis-scripts-core/src/main/java/org/molgenis/script/ScriptUtils.java

```diff
@@ -29,8 +30,7 @@ public static String generateScript(Script script, Map<String, Object> parameter
 		}
 		catch (TemplateException | IOException e)
 		{
-			throw new GenerateScriptException(
-					"Error processing parameters for script [" + script.getName() + "]. " + e.getMessage());
+			throw new InvalidParameterException(script.getName(), e);
```

Comment:
Bit too specific I think, takes two try blocks if you want to determine what step fails (generating the template or processing the parameterValues.